<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>518</width>
    <height>392</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>350</y>
     <width>151</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>475</width>
     <height>53</height>
    </rect>
   </property>
   <property name="title">
    <string>Clog location:</string>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLineEdit" name="clog_location">
      <property name="toolTip">
       <string>export files path</string>
      </property>
      <property name="whatsThis">
       <string>export files path</string>
      </property>
      <property name="clearButtonEnabled">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="getClogFile">
      <property name="maximumSize">
       <size>
        <width>25</width>
        <height>19</height>
       </size>
      </property>
      <property name="toolTip">
       <string>select files to export</string>
      </property>
      <property name="whatsThis">
       <string>select files to export</string>
      </property>
      <property name="text">
       <string>...</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>475</width>
     <height>53</height>
    </rect>
   </property>
   <property name="title">
    <string>Export clog files distination</string>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLineEdit" name="exportClogFilesDistination">
      <property name="toolTip">
       <string>export files path</string>
      </property>
      <property name="whatsThis">
       <string>export files path</string>
      </property>
      <property name="clearButtonEnabled">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="getClogDistinationFiles">
      <property name="maximumSize">
       <size>
        <width>25</width>
        <height>19</height>
       </size>
      </property>
      <property name="toolTip">
       <string>select files to export</string>
      </property>
      <property name="whatsThis">
       <string>select files to export</string>
      </property>
      <property name="text">
       <string>...</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>130</y>
     <width>475</width>
     <height>53</height>
    </rect>
   </property>
   <property name="title">
    <string>Export *.tif files distination</string>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QLineEdit" name="exportTifFilesDistination">
      <property name="toolTip">
       <string>export files path</string>
      </property>
      <property name="whatsThis">
       <string>export files path</string>
      </property>
      <property name="clearButtonEnabled">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="getClogDistinationFiles_2">
      <property name="maximumSize">
       <size>
        <width>25</width>
        <height>19</height>
       </size>
      </property>
      <property name="toolTip">
       <string>select files to export</string>
      </property>
      <property name="whatsThis">
       <string>select files to export</string>
      </property>
      <property name="text">
       <string>...</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="initialEmptyFramesLabel">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>230</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Initial empty frames</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="initialEmptyFrames">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>230</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
  </widget>
  <widget class="QCheckBox" name="timeCounting">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>190</y>
     <width>101</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Time counting</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="finalEmptyFrames">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>230</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
  </widget>
  <widget class="QLabel" name="finalEmptyFramesLabel">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>230</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Initial empty frames</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="numberOfPackets">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>270</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="goodFrames">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>270</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
  </widget>
  <widget class="QLabel" name="goodFramesLabel">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>270</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Good frames</string>
   </property>
  </widget>
  <widget class="QLabel" name="numberOfPacketsLabel">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>270</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Number of packets</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="firstFramesOfAPacket">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>310</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="lastFramesOfAPacket">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>310</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
  </widget>
  <widget class="QLabel" name="lastFramesOfAPacketLabel">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>310</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Last frames of a packet</string>
   </property>
  </widget>
  <widget class="QLabel" name="firstFramesOfAPacketLabel">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>310</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>First frames of a packet</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
